org.gtk.Gtk3theme.elementary.appdata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.gtk.Gtk3theme.elementary</id>
+  <metadata_license>GPL-3.0</metadata_license>
```

Comment:
Perhaps you are confusing this with `<project_license>`, generally people put the metadata (the contents of this file) into the public domain with `<metadata_license>CC0-1.0</metadata_license>`